### DISCUSSION TITLE
Important Announcement

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#83] [Announcements]
<p align="center"><i>This is crossposted from the Discord server.</i></p>

I'm just going to get straight to the point. I haven't really been interacting with any of Redstone Network for a while now. Yes, I'm alive. Yes, I'm okay. I just want to explain why I've been gone for so long, and what's going to be happening to Redstone Network.

For the past 3 years, I have been in a very bad mental state. I genuinely just did not want to do any work at all. I felt like shit, and wanted to do anything else to keep my mind off of my failing grades and stress. Programming was already the thing that helped me do that, so I thought, why not make some stuff to help me get around the firewall at school? It would probably make my life a little easier. And that's basically how Redstone Network was made.

In the process of working on Redstone Network instead of schoolwork, my grades kept falling and I kept feeling worse and worse. Eventually I started taking large hiatuses from Redstone Network because even it couldn't keep me happy. It got especially bad in 8th grade, and nearly brought me to doing some things I would rather not think about.

This week, I talked to my parents and told them about my declining mental health. My dad is a nurse who specializes in psychology stuff, and he basically just told me that I was probably clinically depressed. We're still working on getting me to a doctor to figure out what we're going to do from here.

There are probably going to be some very big changes to Redstone Network soon. I've been thinking about moving Redstone Network leadership (NOT OWNERSHIP) temporarily to someone else soon (don't bother me about it please). If this happens, I will probably be involved in some amount, but I really don't know what extent that will be to. Even if I don't move leadership, I will most likely be taking a very long break (longer than usual) from this group to work on my life. More details will be coming soon.

One more thing: thank you for all the support on Github and other platforms over the years. I couldn't ever dream of making something this popular back when I first started, and I am very grateful.

This isn't the end. Stay tuned.

**Comment by Mr-milky-way** [comment]
Thank you for the update, I hope your mental state improves. You actually inspired me to to work on my own game sites/ways to break the school districts blockers. Thank you.

**Comment by JordanProgramsX** [comment]
**wow, feel better buddy**
